File: src/main/java/org/jukeboxmc/item/ItemBookshelf.java
package org.jukeboxmc.item;

import org.jukeboxmc.block.BlockBookshelf;
import org.jukeboxmc.item.type.Burnable;

import java.time.Duration;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class ItemBookshelf extends Item implements Burnable {

    public ItemBookshelf() {
        super ( "minecraft:bookshelf" );
    }

    @Override
    public BlockBookshelf getBlock() {
        return new BlockBookshelf();
    }

    @Override
    public Duration getBurnTime() {
        return Duration.ofMillis( 300 );
    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/item/behavior/ItemBookshelf.kt
package org.jukeboxmc.server.item.behavior

import org.jukeboxmc.api.item.Burnable
import org.jukeboxmc.api.item.ItemType
import org.jukeboxmc.server.item.JukeboxItem
import java.time.Duration

class ItemBookshelf(itemType: ItemType, countNetworkId: Boolean) : JukeboxItem(itemType, countNetworkId),
    Burnable {

    override fun getBurnTime(): Duration {
        return Duration.ofMillis(300)
    }

}